lib/cyclonedx/cocoapods/bom_builder.rb

```diff
@@ -136,6 +137,16 @@ def bom(version: 1)
                 pod.add_to_bom(xml)
               end
             end
+
+            xml.dependencies do
+              @dependencies.each do |key, array|
```

Comment:
If `dependencies` is nil, this crashes the app.  Easily seen by running the unit tests with `rake spec`.